internal/infrastructure/db/migrations/20230122003454_initial.sql

```diff
@@ -4,7 +4,13 @@ SELECT 'up SQL query';
 
 SET search_path = vehicle_signal_decoding_api, public;
 
-CREATE TYPE trigger_enum as enum ('CAN', 'PID');
+DO $$ 
```

Comment:
let's investigate why these migrations were not working for you, not necessary to do this stuff

Reply:
deleted my edits